make/include/defaults

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/.envrc"
+
+: "${NAMESPACE:=cf}"
+: "${CF_NAMESPACE:=${NAMESPACE}}"
```

Comment:
I don't think we can actually override `CF_NAMESPACE` to something that isn't `${NAMESPACE}`?